README.md

```diff
@@ -147,7 +147,7 @@ class LLMOpenAIAppDelegate: SpeziAppDelegate {
 ```
 
 > [!IMPORTANT]
-> If using `SpeziLLMOpenAI` on macOS, ensure to add the [`Keychain Access Groups` entitlement](https://developer.apple.com/documentation/bundleresources/entitlements/keychain-access-groups) to the enclosing Xcode project via *PROJECT_NAME > Signing&Capabilities > + Capability*. The array of keychain groups can be left empty, only the base entitlement is required.
+> If using `SpeziLLMOpenAI` on macOS, ensure to add the *`Keychain Access Groups` entitlement* to the enclosing Xcode project via *PROJECT_NAME > Signing&Capabilities > + Capability*. The array of keychain groups can be left empty, only the base entitlement is required.
```

Comment:
Same thing here, we should double check these entitlements and what the current state of them is.

Reply:
See comment above.